test/conformance/testdata/conformance.yaml

```diff
@@ -1359,6 +1359,13 @@
     The event is deleted and MUST NOT show up when listing all events.
   release: v1.25
   file: test/e2e/instrumentation/core_events.go
+- testname: Kubelet resource metrics
```

Comment:
These will go away (so no approval from conformance approvers will be needed), when you temporarily demote the test.

Ideally we would have had the tests soaking already...

Reply:
Removed this entry.